linkerd/buffer/src/dispatch.rs

```diff
@@ -44,7 +44,7 @@ pub(crate) async fn run<S, Req, I>(
                                 while let Some(InFlight { tx, .. }) = requests.recv().await {
                                     let _ = tx.send(Err(error.clone().into()));
                                 }
-                                return;
+                                break;
```

Comment:
tioli: but `return` is clearer to me that it's leaving the function scope. `break` makes me have to inspect the parent scopes to figure out whether there are inner loops,e tc.

Reply:
```suggestion
                                return;
```